requirements.txt

```diff
@@ -17,8 +15,8 @@ cython
 git+https://github.com/googlei18n/cu2qu.git
 git+https://github.com/googlei18n/glyphs2ufo.git
 git+https://github.com/googlei18n/ufo2ft.git
-git+https://github.com/robofab-developers/robofab.git
```

Comment:
Defcon should just copy the pens for now!

On Mon, Apr 11, 2016 at 1:57 PM, Cosimo Lupo <notifications@github.com>
wrote:

> In requirements.txt
> <https://github.com/googlei18n/noto-source/pull/5#discussion_r59280008>:
>
> > @@ -17,8 +15,8 @@ cython
> >  git+https://github.com/googlei18n/cu2qu.git
> >  git+https://github.com/googlei18n/glyphs2ufo.git
> >  git+https://github.com/googlei18n/ufo2ft.git
> > -git+https://github.com/robofab-developers/robofab.git
>
> LOL
>
> the problem is always robofab-developers/robofab#52
> <https://github.com/robofab-developers/robofab/issues/52>
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/googlei18n/noto-source/pull/5/files/edcb68ee3c6cb782443eec5d674a24efc8dfa218#r59280008>
>



-- 
behdad
http://behdad.org/